init.lua

```diff
@@ -1,10 +1,29 @@
 local craftguide, datas, npp = {}, {}, 8*3
 
+local group_stereotypes = {
+	["wool"] = "wool:white",
+	["dye"] = "dye:white",
+	["stick"] = "default:stick",
+	["wood"] = "default:wood",
+	["stone"] = "default:cobble",
+	["book"] = "default:book",
+	["sand"] = "default:sand",
+	["tree"] = "default:tree",
+	["leaves"] = "default:leaves",
+	["water_bucket"] = "bucket:bucket_water",
+	["vessel"] = "vessels:glass_bottle",
+	["coal"] = "default:coal_lump",
+	["flower"] = "flowers:dandelion_yellow",
+	["sapling"] = "default:sapling",
+	["mesecon_conductor_craftable"] = "mesecons:wire_00000000_off",
+}
```

Comment:
Some of these groups are already handled in the next statement, i.e. 
```Lua 
elseif minetest.registered_items["default:"..item:sub(7)] then
    item = item:gsub("group:", "default:")
```

So let's not harcode too much in that table, OK ?